hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/TestUtil.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.hive;
+package org.apache.hudi.hive.testutils;
```

Comment:
should we also rename this class `HiveTestUtil` .. optional, feel free to skip

Reply:
Yes `HiveTestUtil` is better... I'll change it in the next PR where I change the remaining modules.